db/schema.rb

```diff
@@ -185,14 +186,14 @@
     t.integer "position"
     t.boolean "latest"
     t.string "full_name"
-    t.integer "size"
     t.string "licenses"
+    t.integer "size"
     t.text "requirements"
     t.string "required_ruby_version"
     t.string "sha256"
     t.hstore "metadata", default: {}, null: false
-    t.datetime "yanked_at"
     t.string "required_rubygems_version"
+    t.datetime "yanked_at"
```

Comment:
please remove the ordering change here and on line 188.

Reply:
Okay. This is the order that Ruby dumped the schema. I didn't specifically seek to modify it other than running the migration. But will change these.

Reply:
This has been resolved.